vault/resource_secrets_sync_gh_destination_test.go

```diff
@@ -77,8 +77,6 @@ resource "vault_secrets_sync_gh_destination" "test" {
   access_token         = "%s"
   repository_owner     = "%s"
   repository_name      = "%s"
-  app_name             = "test-app-name"
```

Comment:
Removed these from this test since these require additional setup in Vault and Secrets Sync for this to work, which was causing the test to fail.